src/core/opentelemetry.ml

```diff
@@ -427,25 +428,24 @@ module Span_ctx : sig
   val of_w3c_trace_context_exn : bytes -> t
   (** @raise Invalid_argument if parsing failed *)
 end = struct
-  (* TODO: trace flags *)
   (* TODO: trace state *)
 
   type t = {
     trace_id: Trace_id.t;
     parent_id: Span_id.t;
-    is_remote: bool;
+    sampled: bool;
```

Comment:
I renamed it to `sampled` because that is what the spec calls it: https://www.w3.org/TR/trace-context/#sampled-flag

Open to any other suggestions though